packages/core/src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -175,36 +224,41 @@ async function getProjectsValidToTransform(mavenJavaProjects: vscode.QuickPickIt
  * As long as the project contains a .java file and a pom.xml file, the project is still considered valid for transformation,
  * and we allow the user to specify the Java version.
  */
-export async function validateOpenProjects(projects: vscode.QuickPickItem[]) {
+export async function validateOpenProjects(projects: vscode.QuickPickItem[], onProjectFirstOpen = false) {
```

Comment:
Is `validateOpenProjects` called anywhere else? Trying to understand why we only log error when `onProjectFirstOpen` is false, but the only call to `validateOpenProjects` set `onProjectFirstOpen` to true.

Reply:
I'm also wondering this @damntrecky